client/control/components/ColumnMenu.js

```diff
@@ -29,6 +29,27 @@ class ColumnMenu extends React.Component {
     </span>
   );
 
+  /**
+   * Given an object, adds necessary properties to
+   * be displayed in a CheckboxList.
+   *
+   * Basically just adds a `value` property from
+   * the col's `slug`. We need this as a function due to some
+   * weird linter issue WRT anon functions returning objects.
```

Comment:
Yay the linter actually saved us here.

I looked in to this some more, and it turns out that `() => { my: 'object' }` is actually a function that executes the statement `my: 'object'`, which is a string (which does nothing), and a label, which is this one weird trick we inherited from C. Not a function that returns an object.